debian/changelog

```diff
@@ -0,0 +1,17 @@
+libndn-cxx (0.5.0-1) unstable; urgency=medium
```

Comment:
Can you make the package version 0? That's important to work around a bug in our jenkins setup, but I also believe 1 is reserved for debian itself, typically.